<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log_options</class>
 <widget class="QDialog" name="Log_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="gb_flags">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cb_readOnly">
        <property name="text">
         <string>readOnly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_acceptRichText">
        <property name="text">
         <string>acceptRichText</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_NoCRLF">
        <property name="text">
         <string>no CRLF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_add_DateTime">
        <property name="text">
         <string>add DateTime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Color">
        <property name="text">
         <string>color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_ErrorAsMessage">
        <property name="text">
         <string>Error as Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_Text_Is_Windows">
        <property name="text">
         <string>is windows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_autosave">
     <property name="title">
      <string>Save log to file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cb_AutoSave">
        <property name="text">
         <string>auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_choice_filename">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_codecs">
     <property name="title">
      <string>Codecs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Locale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_CodecForCStrings"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
